packages/terra-application-navigation/src/common/PopupMenu.module.scss

```diff
@@ -0,0 +1,22 @@
+:local {    
+  .footer {
+    flex: 0 0 auto;
+    position: relative;
+  }
+
+  .content-header {
+    background-color: var(--terra-application-navigation-popup-content-header-background-color, rgba(255, 255, 255, 0.7));
+    border-bottom: var(--terra-application-navigation-popup-content-header-border-bottom, 1px solid #96c2fe);
+  }
+
+  .utility-list {
+    list-style: none;
+    margin: 0;
+    padding-inline-start: 0;
```

Comment:
It doesn't look like this property is supported in IE/Edge. I'd go with `padding: 0;` or `padding-left: 0;`

Reply:
Updated.